src/bnb_trading/validator.py

```
@@ -247,330 +248,392 @@
-        
+
         logger.info(f"Signal Validator инициализиран. Резултати: {results_file}")
-    
+
     def _load_or_create_results(self) -> pd.DataFrame:
         """
         Зарежда съществуващи резултати или създава нов файл
-        
+
         Returns:
             DataFrame с резултатите
```

Ruff (0.12.2)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)